cognite_toolkit/_cdf_tk/builders/_transformation.py

```diff
@@ -19,12 +19,15 @@ class TransformationBuilder(Builder):
     _resource_folder = TransformationLoader.folder_name
 
     def build(
-        self, source_files: list[BuildSourceFile], module: ModuleLocation, console: Callable[[str], None] | None = None
+        self,
+        source_files: list[BuildSourceFile],
+        module: ModuleLocation,
+        console: Callable[[str], None] | None = None,
     ) -> Iterable[BuildDestinationFile | list[ToolkitWarning]]:
         query_files = {
             source_file.source.path: source_file
             for source_file in source_files
-            if source_file.source.path.suffix == ".sql"
+            if source_file.source.path.name.endswith(".sql")
```

Comment:
I don't understand how this is different, and thus solves the bug. If I revert it, the test you made still passes.

Given that you have a file path
```python
my_path = Path('placeholder.Transformation.sql')
my_path.suffix == ".sql"
my_path.name == "placeholder.Transformation.sql"
```
Thus the two expressions will return exactly the same.